SU2_CFD/src/numerics/flow/flow_sources.cpp

```diff
@@ -624,39 +624,62 @@ CNumerics::ResidualType<> CSourceWindGust::ComputeResidual(const CConfig* config
 
   u_gust = WindGust_i[0];
   v_gust = WindGust_i[1];
+// w_gust = WindGust_i[2];
 
   if (GustDir == X_DIR) {
     du_gust_dx = WindGustDer_i[0];
     du_gust_dy = WindGustDer_i[1];
+    //du_gust_dz = WindGustDer_i[2];
     du_gust_dt = WindGustDer_i[2];
```

Comment:
kudos for picking up "feature requests" from CFD online :+1:

Reply:
:-)

Reply:
Did you ever check if the 3D version worked? Otherwise we gained deadcode for no reason...